src/functions/__tests__/lock-session.test.ts

```diff
@@ -0,0 +1,134 @@
+import { makeLockSessionHandler } from "../lock-session";
+import { httpHandlerInputMocks } from "../__mocks__/handlerMocks";
+import * as H from "@pagopa/handler-kit";
+import { aFiscalCode } from "../__mocks__/general";
+import * as E from "fp-ts/lib/Either";
+
+const aValidBody = { fiscal_code: aFiscalCode, unlock_code: "123456789" };
+
+const mockLockUserSession = jest
+  .fn()
+  .mockResolvedValue(E.right({ status: 201 }));
```

Comment:
If the API returns a `no content` maybe the status code should be `204`

Reply:
addressed in https://github.com/pagopa/io-functions-fast-login/pull/19/commits/9a948e0923492e6d126189f32f5fcc5cd4f8ed8a